[RHACS] ROX-21161: Adding new default policy categories (#77751)
openshift/openshift-docs: modules/use-default-security-policies.adoc

* In the {product-title-short} portal, go to *Platform Configuration* -> *Policy Management*.

> The *Policies* view lists the default policies and enables you to configure the policies. 

Reviewer: ```suggestion
The *Policies* view also enables you to configure the policies. 
```
because we said right above there "To view the default policies"
don't need to say it again

* *Severity*: A ranking of the policy, either critical, high, medium, or low, for the amount of attention required.
* *Lifecycle*: The phase of the container lifecycle (build, deploy, or runtime) that this policy applies to, and the phase at which enforcement applies, when the policy is enabled.

The *Policy categories* view lists the categories and enables you to manage the categories for your policies. By default, all the categories are listed. You can optionally filter the categories by using the category name.

The following categories are listed:

* Anomalous Activity
* Cryptocurrency Mining
* DevOps Best Practices
* Docker CIS
* Kubernetes
* Kubernetes Events
* Network Tools
* Package Management
* Privileges
* Security Best Practices
* Supply Chain Security
* System Modification

> * Testing Custom

Reviewer 2: Testing Custom is not a default category, please remove.
